src/fmu/dataio/_oyaml.py

```diff
@@ -1,4 +1,5 @@
 # flake8: noqa
+# type: ignore
```

Comment:
I think this file can be simplified a bit, looks like there is some relics of py27 in here.

Reply:
True, is was borrowed from another library in the python 3.4 days